pom.xml

```diff
@@ -391,6 +392,32 @@
         <scope>provided</scope>
         <type>pom</type>
       </dependency>
+      <dependency>
+        <groupId>io.opencensus</groupId>
+        <artifactId>opencensus-api</artifactId>
+        <version>${opencensus.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.opencensus</groupId>
+        <artifactId>opencensus-impl</artifactId>
```

Comment:
Why we do not use `-impl-lite` or `-impl-core` dependency? Are there any functionality that is not available in either of those that we need?

Reply:
`-impl-lite` [ref](https://www.javadoc.io/doc/io.opencensus/opencensus-impl-lite/latest/io/opencensus/impllite/metrics/MetricsComponentImplLite.html) is android compatible version. 

Official [documentation](https://opencensus.io/quickstart/java/metrics/#0) does recommend `impl` and `api`dependencies. 

Any particular reason to scope it down to -impl-core ?

Reply:
Maybe `-impl-lite` or `-impl-core` have functionality that we need and will not pull in extra dependencies that we do not need?

Reply:
The only extra difference in dependency is the [disruptor](https://mvnrepository.com/artifact/com.lmax/disruptor) framework which seems to improve performance. I am inclined to continue with `-impl` and leverage it